webapp/src/types/api.d.ts

```diff
@@ -12,6 +12,12 @@ export interface APIError {
   message: string;
 }
 
+export interface Competency {
+  id: EntityId;
+  label: string;
+  description: string;
+}
```

Comment:
Will it be necessary to identify the principal_id column of each competency in order for us to implement features such as editing a competency in the future?

Reply:
It would be within the responsibilities of whoever is doing the "Competencies that the current user authored can't be edited #276 " issue/card. But I do believe it would be necessary moving forward.

Reply:
I agree.